subsys/net/ip/tcp2.c

```diff
@@ -1493,7 +1493,7 @@ static struct net_buf *tcp_win_pop(struct tcp_win *w, const char *name,
 	return out;
```

Comment:
In commit "net: tp: Rename tcp_recv() into tp_tcp_recv()" the subject and body contain same information. Better to say in body something like "Because of ...xyz..., rename tcp_recv() into tp_tcp_recv()" or something like that which gives more information.